lib/tests/conftest.py

```diff
@@ -55,30 +57,21 @@ def push_details():
 
 
 @pytest.fixture
-def client(request, monkeypatch, gateway_fqdn, site_parameters, push_details):
-    if request.param == 'authy':
-        yield AuthyClient(
-            api_key=site_parameters['api_key'],
-            api_url=site_parameters['api_url'],
-            poll_interval=0.1,
-            push_details=push_details,
-        )
-    elif request.param == 'starling':
-        tempdir = tempfile.mkdtemp()
-        monkeypatch.setitem(os.environ, 'SCB_PLUGIN_STATE_DIRECTORY', tempdir)
-        monkeypatch.setitem(
-            stable_box_configuration,
-            'starling_join_credential_string',
-            site_parameters['starling_join_credential_string']
-        )
-        yield StarlingClient(
-            environment=site_parameters['environment'],
-            poll_interval=0.1,
-            push_details=push_details
-        )
-        shutil.rmtree(tempdir, True)
-    else:
-        yield None
+def client(monkeypatch, gateway_fqdn, site_parameters, push_details):
+    tempdir = tempfile.mkdtemp()
```

Comment:
we can get rid of tempdir I think , IIRC it was used for testing the file based cache

Reply:
Done